src/gfx/drawable.cpp

```diff
@@ -0,0 +1,73 @@
+#include "drawable.hpp"
+#include <cassert>
+#include <set>
+
+namespace gfx {
+
+#define TRACK_VISITED 1

```

Comment:
is this ok?

Reply:
Ah should've been and NDEBUG here somewhere. fixed that.
Also namespaced it